### PR TITLE
Adding ER-MRL to community project

### PR DESCRIPTION
**Pull request by corentinlger**
## Motivation and Context

- [ ] I have raised an issue to propose this change ([required](https://github.com/DLR-RM/stable-baselines3/blob/master/CONTRIBUTING.md) for new features and bug fixes)

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Documentation (update in the documentation)

## Checklist

- [ ] I've read the [CONTRIBUTION](https://github.com/DLR-RM/stable-baselines3/blob/master/CONTRIBUTING.md) guide (**required**)
- [x] I have updated the changelog accordingly (**required**).
- [ ] My change requires a change to the documentation.
- [ ] I have updated the tests accordingly (*required for a bug fix or a new feature*).
- [ ] I have updated the documentation accordingly.
- [ ] I have opened an associated PR on the [SB3-Contrib repository](https://github.com/Stable-Baselines-Team/stable-baselines3-contrib) (if necessary)
- [ ] I have opened an associated PR on the [RL-Zoo3 repository](https://github.com/DLR-RM/rl-baselines3-zoo) (if necessary)
- [x] I have reformatted the code using `make format` (**required**)
- [x] I have checked the codestyle using `make check-codestyle` and `make lint` (**required**)
- [x] I have ensured `make pytest` and `make type` both pass. (**required**)
- [x] I have checked that the documentation builds using `make doc` (**required**)

Note: You can run most of the checks using `make commit-checks`.

Note: we are using a maximum length of 127 characters per line

### CODE REVIEW

File: docs/misc/projects.rst [hunk 1]
```diff
@@ -239,3 +241,13 @@ Playing Pokemon Red with Reinforcement Learning.
 | Author: Peter Whidden
 | Github: https://github.com/PWhiddy/PokemonRedExperiments
 | Video: https://www.youtube.com/watch?v=DcYLT37ImBY
+
+
+Evolving Reservoirs for Meta Reinforcement Learning
+---------------------------------------------------
+
+A simplified framework to study how neural structures can be optimized at the scale of evolution, in order to enhance the capabilities of agents to learn complex tasks at the scale of development.
```
Comment (araffin) [inline docs/misc/projects.rst:249]: In simpler words, what does that mean/do?

Comment (corentinlger) [reply docs/misc/projects.rst:249]: Mmh it this better ? (Im' sorry if it is not, this is kind of a weird project to explain)

"Meta-RL framework to optimize reservoir-like neural structures (special kind of RNNs), and integrate them to RL agents to improve their training."

Comment (araffin) [reply docs/misc/projects.rst:249]: that's better, maybe specify which aspect is improved?

Comment (corentinlger) [reply docs/misc/projects.rst]: Ok I added that !

**Review by araffin — APPROVED** [review]
LGTM, thanks =)

**Comment by corentinlger** [comment]
Thank you for the lib (;

**Pull request merged**